### PR TITLE
Improved Scrolling for Speakers UI

### PR DESCRIPTION
**Pull request by chhavip**
The required changes have been made with respect to #129 .

### CODE REVIEW

**Comment by chhavip** [comment]
@creativepsyco , the checks are passing it seems, had to restart build once, please review.

File: app/src/main/res/layout/activity_speakers.xml [hunk 1]
```diff
@@ -12,103 +12,127 @@
...
+                    android:layout_marginBottom="2dp"
+                    android:layout_marginTop="2dp"
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:id="@+id/speaker_bio"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:text="Biography"
+                        android:textColor="#000000"
```
Comment (creativepsyco) [inline app/src/main/res/layout/activity_speakers.xml]: color here? I suppose they go into `colors.xml`

File: app/src/main/res/layout/activity_speakers.xml [hunk 2]
```diff
@@ -12,103 +12,127 @@
...
-                        android:id="@+id/imageView_github"
+                    android:layout_marginBottom="2dp"
+                    android:layout_marginTop="2dp"
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:id="@+id/speaker_bio"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:text="Biography"
```
Comment (creativepsyco) [inline app/src/main/res/layout/activity_speakers.xml]: Text as well.

**Comment by creativepsyco** [comment]
Minor comments rest is fine. We can merge it :+1:

**Comment by chhavip** [comment]
@creativepsyco , the text and color was existing code, reformatting is showing it as new, I will put them in respective value files and update the PR

**Comment by creativepsyco** [comment]
Its cool we can merge it anyways no need.

**Comment by chhavip** [comment]
@creativepsyco, okay, let me know if anything else needs to be done.

**Pull request merged**